learning_resources/management/commands/populate_featured_lists.py

```diff
@@ -0,0 +1,93 @@
+"""Management command to create dev-only featured lists for offeror channels"""
+
+import os
+import sys
+
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand
+
+from channels.models import FieldChannel
+from learning_resources.constants import (
+    FEATURED_OFFERORS,
+    LearningResourceRelationTypes,
+    LearningResourceType,
+)
+from learning_resources.models import LearningPath, LearningResource
+from main.utils import now_in_utc
+
+
+class Command(BaseCommand):
+    """create dev-only featured lists for offeror channels"""
+
+    help = "create dev-only featured lists for offeror channelsr"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "resource_count",
+            nargs="?",
+            type=int,
+            default=10,
+            help="Set the number of courses per featured list (default is 10)",
+        )
+
+    def handle(self, *args, **options):  # noqa: ARG002
+        """Create a Favorites userlist for every active user"""
+        if os.environ.get("MITOPEN_ENVIRONMENT") == "production":
```

Comment:
seems like this could be read from settings.ENVIRONMENT instead of directly from the environment

Reply:
good point, updated